### PR TITLE
Update workflow.md

### PR DESCRIPTION
**Pull request by colliand**
CS&S requested changes to the workflow used to create service agreements based on their template. The changes are memorialized here in an updated standard operating procedure.

----
📚 Documentation preview 📚: https://2i2c-team-compass--795.org.readthedocs.build/en/795/

### CODE REVIEW

**Comment by colliand** [comment]
This is a small change to the workflow.md file in the Partnerships section of 2i2c's Team Compass. I request a brief review by @jnywong and, pending satisfaction with the changes, approval of the PR. Thanks in advance Jenny!

File: partnerships/workflow.md [hunk 1]
```diff
@@ -122,11 +122,15 @@ A lead confirmed an upcoming meeting with 2i2c. Calendly sends an automatic mess
...
+2. Return to the Agreements folder and navigate to the [Templates folder](https://drive.google.com/drive/u/1/folders/1oRFLPChp8J9GpgWpnalF74752oNh9tt8), and open the [[TEMPLATE] Outbound Services Agreement shortcut](https://docs.google.com/document/d/1kPgSddJ_Sob0XcTbkDy5UShIAVKPmm04P9ZLsYiOV20/edit#heading=h.qo34o8p9in1e).
+3. In the template document, from the menu bar, select `File > Make a copy`
+4. Name the copy appropriately (i.e. `PartnerX + 2i2c/CS&S - Services Agreement YYYY-MM-DD`)
+5. Click the box below the word "Folder," and in the pop-up menu, the partner folder you navigated to in Step 1 should appear at the top of the list. Mouse-over it and hit the Select button
+6. Do not check either of the boxes (re: sharing or copying comments)
+7. Click Make a Copy - your new agreement will open, and will be filed in the appropriate partner folder.
+8. Set sharing of service agreement (with editor rights) with lead contacts
+9. Add dated entry "YYYY-MM-DD Sent (Draft) Service Agreement as shared Google doc" to `PartnerX >> Running Notes...` file
```
Comment (jnywong) [inline partnerships/workflow.md:133]:
```suggestion
9. Add dated entry "YYYY-MM-DD Sent (Draft) Service Agreement as shared Google doc" to `PartnerX >> Running Notes...` file (create a new one if needed).
```

File: partnerships/workflow.md [hunk 2]
```diff
@@ -122,11 +122,15 @@ A lead confirmed an upcoming meeting with 2i2c. Calendly sends an automatic mess
...
+1. In the [Agreements folder](https://drive.google.com/drive/u/1/folders/1d2N0F0zn3-7wG1FrA_6FHgHtNAyMcJr8), navigate to the folder for the partner you're contracting for (or create a new one if needed) - this step is important for Step 5.
+2. Return to the Agreements folder and navigate to the [Templates folder](https://drive.google.com/drive/u/1/folders/1oRFLPChp8J9GpgWpnalF74752oNh9tt8), and open the [[TEMPLATE] Outbound Services Agreement shortcut](https://docs.google.com/document/d/1kPgSddJ_Sob0XcTbkDy5UShIAVKPmm04P9ZLsYiOV20/edit#heading=h.qo34o8p9in1e).
+3. In the template document, from the menu bar, select `File > Make a copy`
+4. Name the copy appropriately (i.e. `PartnerX + 2i2c/CS&S - Services Agreement YYYY-MM-DD`)
+5. Click the box below the word "Folder," and in the pop-up menu, the partner folder you navigated to in Step 1 should appear at the top of the list. Mouse-over it and hit the Select button
+6. Do not check either of the boxes (re: sharing or copying comments)
+7. Click Make a Copy - your new agreement will open, and will be filed in the appropriate partner folder.
```
Comment (jnywong) [inline partnerships/workflow.md:131]:
```suggestion
7. Click **Make a Copy** - your new agreement will open, and will be filed in the appropriate partner folder.
```

File: partnerships/workflow.md [hunk 3]
```diff
@@ -122,11 +122,15 @@ A lead confirmed an upcoming meeting with 2i2c. Calendly sends an automatic mess
...
-4. Set sharing of service agreement (with editor rights) with lead contacts
-5. Add dated entry "YYYY-MM-DD Sent (Draft) Service Agreement as shared Google doc" to PartnerX >> Running Notes... file
+1. In the [Agreements folder](https://drive.google.com/drive/u/1/folders/1d2N0F0zn3-7wG1FrA_6FHgHtNAyMcJr8), navigate to the folder for the partner you're contracting for (or create a new one if needed) - this step is important for Step 5.
+2. Return to the Agreements folder and navigate to the [Templates folder](https://drive.google.com/drive/u/1/folders/1oRFLPChp8J9GpgWpnalF74752oNh9tt8), and open the [[TEMPLATE] Outbound Services Agreement shortcut](https://docs.google.com/document/d/1kPgSddJ_Sob0XcTbkDy5UShIAVKPmm04P9ZLsYiOV20/edit#heading=h.qo34o8p9in1e).
+3. In the template document, from the menu bar, select `File > Make a copy`
+4. Name the copy appropriately (i.e. `PartnerX + 2i2c/CS&S - Services Agreement YYYY-MM-DD`)
+5. Click the box below the word "Folder," and in the pop-up menu, the partner folder you navigated to in Step 1 should appear at the top of the list. Mouse-over it and hit the Select button
```
Comment (jnywong) [inline partnerships/workflow.md:129]:
```suggestion
5. Click the box below the word "Folder," and in the pop-up menu, the partner folder you navigated to in Step 1 should appear at the top of the list. Mouse-over it and hit the **Select** button
```

**Review by jnywong — APPROVED** [review]

**Comment by choldgraf** [comment]
Feel free to just merge these @jnywong - as a general rule, anything within the Partnerships folder should be mergeable by anybody in Partnerships, unless @colliand requests otherwise.

**Pull request merged**